Add notice on swerve support (#1529)
wpilibsuite/frc-docs: source/docs/software/pathplanning/index.rst

Path Planning is the process of creating and following trajectories. These paths use the WPILib trajectory APIs for generation and a :ref:`Ramsete Controller <docs/software/advanced-controls/trajectories/ramsete:Ramsete Controller>` for following. This section highlights the process of characterizing your robot for system identification, trajectory following and usage of PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for additional information about the API and non-commandbased usage.

Notice on Swerve Support
------------------------

> Swerve support in path following has a couple of issues that teams need to be aware of:

Reviewer: ```suggestion
Swerve support in path following has a couple of limitations that teams need to be aware of:
```

- WPILib currently does not support swerve in simulation, please see `this <https://github.com/wpilibsuite/allwpilib/pull/3374>`__ pull request.

> - SysID only supports tuning the swerve heading using a simple motor project and is unable to collect module velocity data.

Reviewer 2: "SysID only supports tuning the swerve heading"
Is heading the best word to use here? I assume you mean the swerve azimuth or tuning the swerve turning motor. Also, is a simple motor project still a thing in SysID? To my knowledge, it is just general mechanism now.

> - Pathweaver and Trajectory following currently do not take heading into account. Path following using the WPILib trajectory framework on swerve will be the same as a DifferentialDrive robot.

Reviewer: ```suggestion
- Pathweaver and Trajectory following currently do not incorporate independent heading. Path following using the WPILib trajectory framework on swerve will be the same as a DifferentialDrive robot.
```

> We are sorry for the inconvenience at this time.

Reviewer: ```suggestion
We are sorry for the inconvenience.
```
